templates/repo/branch/list.tmpl

```diff
@@ -193,4 +225,27 @@
 		</form>
 	</div>
 </div>
+
+<div class="ui g-modal-confim modal" id="rename-branch-modal">
+	<div class="header">
+		{{.locale.Tr "repo.settings.rename_branch"}}
+	</div>
+	<div class="content">
+		<form class="ui form" id="rename-branch-form" action="{{$.Repository.Link}}/settings/rename_branch" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="field">
+				<label>
+					<span class="text" id="modal-rename-branch-to-span" data-rename-branch-to="{{.locale.Tr "repo.branch.rename_branch_to"}}"></span>
+				</label>
+			</div>
+			<input id="from" name="from" value="" type="hidden" required>
+			<div class="required field">
+				<input id="to" name="to" required>
+			</div>
+			<div class="center">
```

Comment:
Not sure why it's `center`. I would prefer to keep the same style as other modal buttons: right aligned.

Reply:
commit: [67bf622](https://github.com/go-gitea/gitea/pull/24380/commits/67bf622691c17a91fd9c2ae8bc051317e7de6e64).

buttons are moved to the right now.

---
 
For a uniform style for this page, the buttons in create brach dialog are changed to the "base/modal_actions_confirm" template too.

![image](https://user-images.githubusercontent.com/33891828/235277534-88e5a425-f2b8-405b-ac5f-6aa9e0980a80.png)